shared/utils/enable-sl.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python2
+
+"""
+Takes given XCCDF or DataStream and adds Scientific Linux CPE name next to RHEL CPE names.
+Can automatically recognize RHEL5, 6, 7 CPEs and adds the Scientifc Linux ones next to
+those accordingly.
+
+Apart from adding the CPEs it adds a notice informing the user that the content
+has been enabled for Scientifc Linux and what are the implications.
+
+Author: Martin Preisler <mpreisle@redhat.com>
+"""
+
+from xml.etree import cElementTree as ElementTree
+import sys
+
+XCCDF11_NS = "http://checklists.nist.gov/xccdf/1.1"
+XCCDF12_NS = "http://checklists.nist.gov/xccdf/1.2"
+
+RHEL_SL_CPE_MAPPING = {
+    "cpe:/o:redhat:enterprise_linux:4": "cpe:/o:scientific:linux:4",
+    "cpe:/o:redhat:enterprise_linux:5": "cpe:/o:scientific:linux:5",
+    "cpe:/o:redhat:enterprise_linux:6": "cpe:/o:scientific:linux:6",
+    "cpe:/o:redhat:enterprise_linux:7": "cpe:/o:scientific:linux:7",
+}
+
+SL_NOTICE = \
+    "<div xmlns=\"http://www.w3.org/1999/xhtml\">\n" \
+    "<p>This benchmark is a direct port of a <i>SCAP Security Guide </i> " \
+    "benchmark developed for <i>Red Hat Enterprise Linux</i>. It has been " \
+    "modified through an automated process to remove specific dependencies " \
+    "on <i>Red Hat Enterprise Linux</i> and to function with <i>Scientifc Linux</i>. " \
+    "The result is a generally useful <i>SCAP Security Guide</i> benchmark " \
+    "with the following caveats:</p>\n" \
+    "<ul>\n" \
+    "<li><i>Scientifc Linux</i> is not an exact copy of " \
+    "<i>Red Hat Enterprise Linux</i>. There may be configuration differences " \
+    "that produce false positives and/or false negatives. If this occurs " \
+    "please file a bug report.</li>\n" \
+    "\n" \
+    "<li><i>Scientifc Linux</i> has its own build system, compiler options, patchsets, " \
+    "and is a community supported, non-commercial operating system. " \
+    "<i>Scientifc Linux</i> does not inherit " \
+    "certifications or evaluations from <i>Red Hat Enterprise Linux</i>. As " \
+    "such, some configuration rules (such as those requiring " \
+    "<i>FIPS 140-2</i> encryption) will continue to fail on <i>Scientifc Linux</i>.</li>\n" \
+    "</ul>\n" \
```

Comment:
We can update the caveats using language pulled from the Scientific Linux wikipedia:

`````
- Scientific Linux is not an exact copy of Red Hat Enterprise Linux. Scientific Linux is a Linux distribution produced by
Fermi National Accelerator Laboratory. It is a free and open source operating system based on Red Hat Enterprise Linux
and aims to be "as close to the commercial enterprise distribution as we can get it." There may be configuration differences
that produce false positives and/or false negatives. If this occurs, please file a bug report.

- Scientific Linux is derived from the free and open source software made available by Red Hat, but it is not produced, maintained or supported by Red Hat. Scientific Linux has its own build systems, compiler options, patchsets, packages, and is a community supported, non-commercial operating system. Scientific Linux does not inherit certifications or evaluations from Red Hat Enterprise Linux. As such, some configuration rules (such as those requiring FIPS 140-2 encryption) will continue to fail on Scientific Linux.
`````

ref http://en.wikipedia.org/wiki/Scientific_Linux